views/partials/patient-name.handlebars

```diff
@@ -0,0 +1,9 @@
+{{#each patients as |patient| }}
```

Comment:
We might need these each statements to wrap the partial statement in homepage (i.e. that's how I've always configured it) -- but I'm not sure this _wouldn't_ work, so just noting in case